code.js

```diff
@@ -76,6 +76,20 @@ Mergesort
 
 */
 
+// for testing purposes im using this recursive sort function from a Lab in cosc 2030, 
+function mergeSort(arr,left,right,size)
+{
+  if(left<right)
```

Comment:
Do you need this  condition here? How could you make the code work in the other case?

Reply:
yes, because we need a base case for recursion. However just as 'left < right' must be the condition to follow, so could 'right > left'.